models/StringLib_Test/SW.aadl

```diff
@@ -0,0 +1,237 @@
+package SW
+public
+	with Data_Model;
+	with Base_Types;
+	with CASE;
```

Comment:
The CASE package is not part of the smaccm repo.  Could you include it, so there aren't external dependencies?

Reply:
Done.